src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/xlf/VSPackage.zh-Hant.xlf

```diff
@@ -69,7 +69,7 @@
       </trans-unit>
       <trans-unit id="24">
         <source>F# Project Files (*.fsproj);*.fsproj</source>
-        <target state="translated">F# 專案檔 (*.fsproj); *.fsproj</target>
+        <target state="new">F# Project Files (*.fsproj);*.fsproj</target>
```

Comment:
Another potential regression.

Reply:
Thank you for spotting this.
I logged ticket as this seems as tool issue.

Reply:
We updated PR not to block merge